<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_1">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Введите минимальную широту</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="min_lat_input">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Введите максимальную широту</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="max_lat_input">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_11">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_11">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Введите минимальную долготу</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="min_lon_input">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>110</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_12">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Введите максимальную долготу</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="max_lon_input">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>220</y>
      <width>192</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="topMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="apply_filter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>30</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>85</red>
             <green>255</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>85</red>
             <green>255</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>85</red>
             <green>255</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="text">
        <string>Фильтр</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>1441</width>
      <height>771</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_13">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>30</number>
     </property>
     <item row="0" column="0">
      <widget class="QTableWidget" name="airport_table">
       <property name="rowCount">
        <number>23</number>
       </property>
       <property name="columnCount">
        <number>4</number>
       </property>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>345</number>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <column/>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
